### ISSUE TITLE
[Functions][Regression] Cardinality is missing for EventHubTrigger

### ISSUE THREAD

**Issue by pragnagopa** [Microsoft/azure-maven-plugins#561]
### Plugin name and version
azure.functions.maven.plugin
1.2.1
### Expected behavior
EventHubTrigger with following signature:
```
 @EventHubTrigger(name = "messages", eventHubName = "test-input-java", connection = "AzureWebJobsEventHubSender", dataType = "string", cardinality = Cardinality.MANY) String[] messages,
```
should generate function.json with Cardinality Many
```
{
    "type" : "eventHubTrigger",
    "name" : "messages",
    "direction" : "in",
    "dataType" : "string",
    "consumerGroup" : "$Default",
    "connection" : "AzureWebJobsEventHubSender",
    "cardinality" : "many",
    "eventHubName" : "test-input-java"
  }
```
### Actual behavior
in Version 1.2.1 , Cardinality is missing

**Comment by pragnagopa** [comment]
@Flanker32 / @jdneo - This is a regression and needs to be addressed ASAP.

**Comment by Flanker32** [comment]
@pragnagopa  For now we write values differs from its default value to function.json as we don't know whether the value is set by users, or is there anyway for maven tooling to find whether an attribute is required in function.json like schema or annotation?

**Comment by Flanker32** [comment]
FIxed in #562
